holesky/run.sh

```diff
@@ -3,6 +3,63 @@
 
 . ./.env
 
+# Initialize variables
+NODE_ECDSA_KEY_FILE_HOST=""
+NODE_ECDSA_KEY_PASSWORD=""
+OPERATION_TYPE=""
+QUORUMS=""
+
+# Loop through command-line arguments
+# shellcheck disable=SC2039
+while [[ $# -gt 0 ]]; do
```

Comment:
I hope this is tested!

Reply:
yea I tested it lol - I will do more testing - does bash file has unit test ? :P I can try to find

Reply:
did tests of operations - works pretty good.